app/views/profiles/show.html.erb

```diff
@@ -3,4 +3,4 @@
 <h4>Zip Code: <%= @user.zip_code %></h4>
 <%= link_to "Edit Profile", "/profile/#{@user.id}/edit" %>
 <br><br><br>
-<%= link_to "Delete Profile", "/profile/#{@user.id}", method: :delete %>
+<%= link_to "Delete Profile", "/profile/#{@user.id}", method: :delete, data: {confirm: "Are you sure?"}%>
```

Comment:
learning every day, I didn't know these messages could be added to a link_to helper